cfn-resources/network-container/cmd/resource/create.go

```diff
@@ -41,19 +39,18 @@ func createOperation(req handler.Request, prevModel *Model, currentModel *Model)
 		}, err
 	}
 
-	// Create atlas client
-	if currentModel.Profile == nil || *currentModel.Profile == "" {
-		currentModel.Profile = aws.String(profile.DefaultProfile)
-	}
-	client, peErr := util.NewMongoDBClient(req, currentModel.Profile)
+	util.SetDefaultProfileIfNotDefined(&currentModel.Profile)
+	client, peErr := util.NewAtlasClient(&req, currentModel.Profile)
 	if peErr != nil {
 		return *peErr, nil
 	}
 
-	containerRequest := &mongodbatlas.Container{}
-	containerRequest.RegionName = *currentModel.RegionName
-	containerRequest.ProviderName = constants.AWS
-	containerRequest.AtlasCIDRBlock = *currentModel.AtlasCidrBlock
+	aws := constants.AWS
```

Comment:
any specific reason to define the variable here? Can we just use `constants.AWS`?

Reply:
unfortunately no 😢 , `ProviderName` requires a pointer and you cannot use `&` with a constant

Reply:
possible aleternative is to use admin.PtrString(constants.AWS)

Reply:
ah i was not aware of this function. Updating

Reply:
Updated